VRDR.Messaging/BaseMessage.cs

```diff
@@ -76,31 +76,35 @@ public BaseMessage(String messageType)
         }
 
         /// <summary>Helper method to return a XML string representation of this DeathRecordSubmission.</summary>
+        /// <param name="prettyPrint">controls whether the returned string is formatted for human readability (true) or compact (false)</param>
```

Comment:
Seems a bit odd to add pretty printing functionality here, but not in other parts of the library, what's the goal?

Reply:
Goal was to make it easier to create human-readable example messages for testing. Currently have to find a separate JSON/XML formatter, to pretty print the condensed JSON/XML output.

Reply:
If you don't already use it I recommend jq